### ISSUE TITLE
Save as csv

### ISSUE THREAD

**Issue by Ubaid93** [Mottl/GetOldTweets3#51]
Hi How can we save the output in csv format in python? instead of just printing it

**Comment by marquisvictor** [comment]
I think we would have to use Twitter API and Tweepy to get that info.

**Comment by marquisvictor** [comment]
But we can't get old tweets using that method. I wish it were possible here

**Comment by parayamelo** [comment]
Same issue here. The csv has a "geo" column but it is always empty. @Mottl do you know the issue?

**Comment by Mottl** [comment]
It seems like Tweeter is no more sharing a geo location within the `<span class="Tweet-geo" ...>` tag. There is no such a tag in `https://twitter.com/i/search/timeline` data which is used for tweets retrieval.

**Comment by parayamelo** [comment]
I see, so the only way would be to pass the near argument to actually get tweets from a particular region/area.

**Comment by Mottl** [comment]
Yes, but what if you want to collect all the tweets on the given subject?..

**Comment by parayamelo** [comment]
Exactly, that is the problem I am facing right now, it is either one or the other, apparently.... What do you recommend?

**Comment by Mottl** [comment]
Use the official Tweeter API which I think has this capability.

**Comment by parayamelo** [comment]
I agree but then we are stuck with the 7 days limit.... Anyway, thanks for the tool! I like how easy it is to use!

**Comment by badaouisaad** [comment]
Dear Mottl,

I am facing similar issue where the column "geo" is always empty and i saw your response  when you said 

**"It seems like Tweeter is no more sharing a geo location within the <span class="Tweet-geo" ...> tag. There is no such a tag in https://twitter.com/i/search/timeline data which is used for tweets retrieval."**

However, does it mean that we can not filter tweets by locations using  got.manager.TweetCriteria().setQuerySearch("#BLM").setNear("Minnesota, USA").setWithin("1000km")  ?

Are two issues "geocode" tag and "setnear" related to each other?

Thanks very much

**Comment by badaouisaad** [comment]
Any comment on my last comment? Thank you
